Clinical-Genomics/cg
# Fix/check finished mip in tb

## Description
This PR fixes a bug implemented with https://github.com/Clinical-Genomics/cg/pull/1151. If a user set a case to finished we will did not alter the status in the sample info file. Since we moved away from reading analysis status from the sample info file we will instead go back to checking TB status.

### How to prepare for test
`bash /home/proj/production/servers/resources/hasta.scilifelab.se/update-cg-stage.sh fix/check-finished-mip-in-tb`

### How to test
- [ ] copy cg-prod to stage
- [ ] ensure that sampleinfo for case is set to `not finished`
- [ ] Set case to completed in trailblazer
- [ ] do `cg workflow mip-dna  store-available`

### Expected test outcome
- [ ] check that analysis is fully stored

## Review
- [ ] tests executed by @karlnyr 
Thanks for filling in who performed the code review and the test!

### This [version](https://semver.org/) is a

## Implementation Plan
- [ ] Deploy this branch
